examples/volume_slice1.py

```diff
@@ -0,0 +1,67 @@
+"""
+Render slices through a volume, by uploading to a 2D texture.
+Simple and ... slow.
+"""
+
+import imageio
+import visvis2 as vv
+
+from PyQt5 import QtWidgets
+from wgpu.gui.qt import WgpuCanvas
+
+
+class WgpuCanvasWithScroll(WgpuCanvas):
+    def wheelEvent(self, event):  # noqa: N802
+        degrees = event.angleDelta().y() / 8
+        scroll(degrees)
+
+
+app = QtWidgets.QApplication([])
+
+canvas = WgpuCanvasWithScroll()
+renderer = vv.renderers.WgpuRenderer(canvas)
+scene = vv.Scene()
+
+vol = imageio.volread("imageio:stent.npz")
+nslices = vol.shape[0]
+index = nslices // 3
+im = vol[index].copy()
+
+tex = vv.Texture(im, dim=2, usage="sampled")
+
+geometry = vv.PlaneGeometry(200, 200, 12, 12)
+material = vv.MeshBasicMaterial(map=tex.get_view(filter="linear"), clim=(0, 255))
+plane = vv.Mesh(geometry, material)
+plane.scale.y = -1
+scene.add(plane)
+
+fov, aspect, near, far = 70, -16 / 9, 1, 1000
+camera = vv.PerspectiveCamera(fov, aspect, near, far)
+camera.position.z = 200
+
+
+def scroll(degrees):
+    global index
+    index = index + int(degrees / 15)
+    index = max(0, min(nslices - 1, index))
+    im = vol[index]
+    tex.data[:] = im
+    tex.update_range((0, 0, 0), tex.size)
+    material.dirty = 1
```

Comment:
Why is this needed?

Reply:
because of how the renderer works right now. That process should be made somewhat smarter.